ewebsock/Cargo.toml

```diff
@@ -39,11 +39,12 @@ tokio = [
 
 [dependencies]
 document-features = "0.2"
+http = "1.0.0"
 log = "0.4"
 
 # native:
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
-tungstenite = { version = ">=0.17, <=0.20" }
+tungstenite = { version = ">=0.17, <=0.21" }
```

Comment:
this needs to be set to the version number of the first tungstenite release with the `with_header` function

Reply:
Yes you're right !
I'm waiting for the new version. I'm tagging the PR as DRAFT to wait for it